gensim/models/experimental/UI_Example.ipynb

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
As I already suggest
- should be executable always (use `gensim-data` for vectors + download explicitly train data)
- add small evaluation (comparing with w2v, fasttext)
- more examples (like in the end of notebook)
- add pre-processing on "user-side"